open_city_profile/graphene.py

```diff
@@ -178,3 +183,55 @@ def __init_subclass_with_meta__(
             _meta=_meta,
             **options,
         )
+
+
+def send_allowed_data_field_sentry_warning(
+    message: str, service: Optional[Service], field_name: str
+):
+    with sentry_sdk.push_scope() as scope:
+        scope.set_extra("service", getattr(service, "name", "Service not identified"))
+        scope.set_extra("field_name", field_name)
+
+        sentry_sdk.capture_message(message, level="warning")
+
+
+class AllowedDataFieldsMiddleware:
+
+    def _is_profile_query(self, info):
+        return any(
+            selection.name.value in ("myProfile", "profile", "profiles")
+            for selection in info.operation.selection_set.selections
+        )
+
+    def resolve(self, next, root, info, **args):
```

Comment:
I know the documentation says `**args`, but maybe for consistency we should use `**kwargs` here.

Reply:
Very good proposion. Didn't notice it myself - obviously 😅